schemas/internal/command.proto

```diff
@@ -39,6 +39,7 @@ message Command {
         CancelUserTaskRunRequest cancel_user_task = 19;
         BulkUpdateJob bulk_job = 20;
         TaskAttemptRetryReady task_attempt_retry_ready = 21;
+        DeletePrincipalRequest delete_principal = 22;
```

Comment:
`DeletePrincipalRequest` is an external request. why it needs to be in this proto file?

Reply:
Because it's a `SubCommand`. This is the protobuf for a `Command`.

We have other "external requests" which are also `SubCommand`s, like `RunWfRequest` and `DeleteWfRunRequest`.

However, you're right...it's a _METADATA_ command, not a run command, so it shouldn't be here. That was a brain fart.